src/transformers/pipelines.py

```diff
@@ -2631,6 +2641,49 @@ def _concat_inputs_history(self, inputs: List[List[int]], histories: List[Option
 }
 
 
+def check_task(task: str) -> Tuple[Dict, Any]:
+    """
+    Checks an incoming task string, to validate it's correct and return the
+    default Pipeline and Model classes, and default models if they exist.
+
+    Args:
+        task (:obj:`str`):
+            The task defining which pipeline will be returned. Currently accepted tasks are:
+
+            - :obj:`"feature-extraction"`
+            - :obj:`"sentiment-analysis"`
+            - :obj:`"ner"`
+            - :obj:`"question-answering"`
+            - :obj:`"fill-mask"`
+            - :obj:`"summarization"`
+            - :obj:`"translation_xx_to_yy"`
+            - :obj:`"translation"`
+            - :obj:`"text-generation"`
+            - :obj:`"conversation"`
```

Comment:
```suggestion
            - :obj:`"conversational"`
```

from https://huggingface.co/models?filter=conversational